From a9c5a5d98c12cf03e60dbcc61744a154d0da54a1 Mon Sep 17 00:00:00 2001
From: Vincent Abriou <vincent.abriou@st.com>
Date: Fri, 6 Dec 2019 10:34:39 +0100
Subject: [PATCH 18/31] media: ov5640: fix JPEG regression

Regression introduced by commit 7cb013b1049a2a599c1fc344644528eb8bf985b8
media: ov5640: Set JPEG output timings when outputting JPEG data

disable this change...

Signed-off-by: Hugues Fruchet <hugues.fruchet@st.com>
---
 drivers/media/i2c/ov5640.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/media/i2c/ov5640.c b/drivers/media/i2c/ov5640.c
index 554106e..d5d8b1a 100644
--- a/drivers/media/i2c/ov5640.c
+++ b/drivers/media/i2c/ov5640.c
@@ -1597,11 +1597,13 @@ static int ov5640_set_timings(struct ov5640_dev *sensor,
 {
 	int ret;
 
+#if 0
 	if (sensor->fmt.code == MEDIA_BUS_FMT_JPEG_1X8) {
 		ret = ov5640_set_jpeg_timings(sensor, mode);
 		if (ret < 0)
 			return ret;
 	}
+#endif
 
 	ret = ov5640_write_reg16(sensor, OV5640_REG_TIMING_DVPHO, mode->hact);
 	if (ret < 0)
-- 
2.7.4

